integration-tests/modules/__tests__/cart/store/carts.spec.ts

```diff
@@ -65,7 +65,7 @@ medusaIntegrationTestRunner({
       })
 
       describe("POST /store/carts", () => {
-        it("should create a cart", async () => {
+        it.skip("should create a cart", async () => {
```

Comment:
Let's just follow-up and re-enable these, but it shouldn't be a blocker IMO

Reply:
yeah it is not for that pr, I am investigating it but if there is an issue it is separate

Reply:
Agree, would also just get this in now, and resolve (future) flaky tests in follow-up PRs